corehq/apps/api/resources/v0_4.py

```diff
@@ -120,6 +120,7 @@ def obj_get_list(self, bundle, domain, **kwargs):
 
         def wrapper(doc):
             if doc['doc_type'] in xform_doc_types:
+                doc.pop('user_id', None)  # needed when xform is stored in sql
                 return xform_doc_types[doc['doc_type']].wrap(doc)
```

Comment:
agree with the consistency bit, but this bit looks suspricious:

```
xform_doc_types[doc['doc_type']].wrap(doc)
```
isn't that saying wrap a sql form as a couch form?

agree probably best not to wrap